packages/plugin-flex/src/clients/ServerlessClient.ts

```diff
@@ -156,6 +152,40 @@ export default class ServerlessClient {
     return build;
   }
 
+  /**
+   * Deletes the {@link EnvironmentInstance}
+   * @param serviceSid   the service sid
+   * @param environment  the environment sid
+   */
+  async deleteEnvironment(serviceSid: string, environmentSid: string): Promise<boolean> {
+    const service = await this.getService(serviceSid);
+    if (!service) {
+      return false;
+    }
+
+    return service.environments().get(environmentSid).remove();
+  }
+
+  /**
+   * Returns the {@link EnvironmentInstance}
+   * @param serviceSid  the service sid
+   * @param pluginName  the plugin name
+   */
+  async getEnvironment(serviceSid: string, pluginName: string): Promise<EnvironmentInstance | null> {
+    const service = await this.getService(serviceSid);
+    if (!service) {
+      return null;
+    }
+
+    const list = await service.environments().list();
+    const environment = list.find((e) => e.uniqueName === pluginName);
+    if (!environment || !environment.sid) {
```

Comment:
Why do you need to do this check? Your return type is `EnvironmentInstance | null` already - so can't you just `return list.find((e) => e.uniqueName === pluginName);`?

Reply:
i could, but i would have to do that check in all the functions that i'm calling this method